_rules/device-motion-user-interface-7677a9.md

```diff
@@ -0,0 +1,318 @@
+---
+id: 7677a9
+name: Device motion can be operated by user interface
```

Comment:
I don't think this title works... You mean "device motion functionality can also be controlled from the user interface"? The current phrasing sounds like you want the UI to be able to move the device.